metals/src/main/scala/scala/meta/internal/metals/WorkspaceLspService.scala

```diff
@@ -215,7 +235,7 @@ class WorkspaceLspService(
     clientConfig,
     shellRunner,
     clientConfig.icons,
-    folders.head.path,
+    fallbackService.path,
```

Comment:
Will this also not create fallbackService eagerly?

Reply:
I think I fixed all now.